app/src/sharedTest/java/org/oppia/app/utility/RatioExtensionsTest.kt

```diff
@@ -0,0 +1,40 @@
+package org.oppia.app.utility
+
+import android.content.Context
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.common.truth.Truth.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.model.RatioExpression
+import org.robolectric.annotation.Config
+import org.robolectric.annotation.LooperMode
+
+/** Tests for [RatioExtensions]. */
+@RunWith(AndroidJUnit4::class)
+@LooperMode(LooperMode.Mode.PAUSED)
+@Config(manifest = Config.NONE)
+class RatioExtensionsTest {
+
+  private lateinit var context: Context
+
+  @Before
+  fun setUp() {
+    context = ApplicationProvider.getApplicationContext()
+  }
+
+  @Test
+  fun testRatio_ratio_returnAccessibleRatioString() {
```

Comment:
This is testing 3 elements. I suggest also adding one for 2 to ensure the string is correctly being computed for different lengths of ratios. Please make sure the test names reflect the differences, too.

Reply:
done